modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/get_top.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const SIMULATION_DEVICE_IFRAME = 'simulationDeviceIframe';
+let _topWindow;
```

Comment:
Is there a work to be done at this variable? At least a start value? If not, why not just leaving `let topWindow;` inside function's closure?

Reply:
It's outside the function's scope, it was inside previously 😄

Reply:
@diegonvs the previous implementation used a global variable on `Util._topWindow`. Since we no longer want to store private variables on that object, we use a local variable to this file instead. That way if this function is invoked twice, it will re-use that variable on the second invocation.

Reply:
Looks like we commented at the exact same time 😆

Reply:
<img src="https://media1.giphy.com/media/xT0xex57qDRDwyyqyc/giphy.gif"/>